components/Schedule/SportClassCard.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+
+export default function SportClassCard({ classData }) {
```

Comment:
you can destructure the classData further like so "export default function SportClassCard({ classData: {name, startTime, teacher, duration }  }) {"

This avoids have to write classData repeatedly.

Reply:
thanks, i either way have to repeat one or the other, i prefer the current option BUT will definitely keep this in mind for future where the repetition is more out of hand.